### PR TITLE
[DRAFT] Translations - blog posts, docs [ES, JA, PT, RU, ZH, KO]

### PR DESCRIPTION
**Pull request by ilyaspiridonov**
Translated by Alconost
Includes translations of blog posts and documentation, as well as some fixes in existing ES translations.

### CODE REVIEW

**Comment by jpmedley** [comment]
@robdodson

**Review by jpmedley — APPROVED** [review]

**Review by devnook — CHANGES_REQUESTED** [review]
@ilyaspiridonov Can you please fix lint warnings?

**Comment by ilyaspiridonov** [comment]
@devnook Lint tests seem to be in green now

**Comment by devnook** [comment]
Hi @ilyaspiridonov ! The build is still broken. I'm taking a look now.

File: src/site/content/ko/blog/defining-core-web-vitals-thresholds/index.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+
+## 다시 짚어보기: Core Web Vitals 메트릭 및 임계값
+
+2020년 Core Web Vitals은 Large Contentful Paint(최대 콘텐츠풀 페인트, LCP), First Input Delay(최초 입력 지연, FID), Cumulative Layout Shift(누적 레이아웃 이동, CLS) 메트릭입니다. 각 메트릭은 사용자 경험의 다른 측면을 측정합니다. LCP는 인지된 로드 속도를 측정하고 페이지 로드 타임라인에서 페이지의 메인 콘텐츠가 로드된 지점을 표시합니다. FID는 응답성을 측정하고 사용자가 페이지와 처음 상호 작용을 시도할 때 느끼는 경험을 수량화합니다. CLS는 시각적 안정성을 측정하고 시각적 페이지 콘텐츠의 예기치 못한 레이아웃 이동을 수치화하여 표현합니다.
+
+각 Core Web Vitals 메트릭에는 성능을 "양호(good)", "개선 필요(needs improvement)" 또는 "나쁨(poor)"으로 분류하는 관련 임계값이 있습니다.
+
+<div class="w-stack w-stack--center w-stack--md">{% Img src="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/ZZU8Z7TMKXmzZT2mCjJU.svg", alt="최대 콘텐츠풀 페인트 권장 임계값", width="400", height="350" %} {% Img src="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/iHYrrXKe4QRcb2uu8eV8.svg", alt="최초 입력 지연 권장 임계값", width="400", height="350" %} {% Img src="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/dgpDFckbHwwOKdIGDa3N.svg", alt="누적 레이아웃 이동 권장 임계값", width="400", height="350" %}</div>
```
Comment (devnook) [inline src/site/content/ko/blog/defining-core-web-vitals-thresholds/index.md:25]: These images are duplicated, I think. compare to: https://raw.githubusercontent.com/GoogleChrome/web.dev/main/src/site/content/en/blog/defining-core-web-vitals-thresholds/index.md

Comment (devnook) [inline src/site/content/ko/blog/defining-core-web-vitals-thresholds/index.md:25]:
```suggestion
```

File: src/site/content/ko/blog/optimize-lcp/index.md [hunk 2]
```diff
@@ -0,0 +1,439 @@
...
+다음 차트는 이 패턴을 사용하여 사이트에서 LCP 분포가 어떻게 감소했는지 보여줍니다.
+
+<figure class="w-figure">
+  {% Img
+    src="image/admin/uB0Sm56R88MRF16voQ1k.png",
+    alt="HTML 캐싱 전후 최대 콘텐츠풀 페인트 분포",
+    width="800",
+    height="495"
+  %}
+  <figcaption class="w-figcaption">페이지 로드 시 서비스 작업자 여부에 따른 최대 콘텐츠풀 분포 - <a href="https://philipwalton.com/articles/smaller-html-payloads-with-service-workers/">philipwalton.com</a></figcaption>
```
Comment (devnook) [inline src/site/content/ko/blog/optimize-lcp/index.md:87]:
```suggestion
```

**Review by devnook — APPROVED** [review]

**Pull request merged**